src/main/java/cat/udl/eps/softarch/demo/domain/Schedule.java

```diff
@@ -0,0 +1,31 @@
+package cat.udl.eps.softarch.demo.domain;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.sql.Time;
+
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class Schedule {
+
+    @Id
+    @GeneratedValue
+    private int Id;
+
+    @NotNull
+    @NotBlank

```

Comment:
Hi @pautorne, revising the Schedule entity, I see that you don't use properly the annotations not blank and not null.

NotNull: This annotation is typically used to specify that a field must not be null. It's a general constraint that applies to any type of object.

@NotBlank: This annotation, on the other hand, is more specific. It's typically used for string fields and indicates that the annotated string must not be null and must contain at least one non-whitespace character.

In this case of DateTime fields, it's not good practice to use @NotBlank, because DateTime objects are not directly comparable to strings.

Reply:
Done